### PR TITLE
Renaming custom panels to Observability Dashboards

### PR DESCRIPTION
**Pull request by mengweieric**
### Description
Observability Dashboards renaming for toast message.

### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [x] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by ps48 — APPROVED** [review]

File: test/panels_constants.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@
  */
 
 export const panelBreadCrumbs = [
-  { text: 'Operational panels', href: '#/operational_panels/' },
+  { text: 'Observability Dashboards', href: '#/operational_panels/' },
```
Comment (derek-ho) [inline test/panels_constants.tsx:7]: should we change the href too?

**Review by derek-ho — APPROVED** [review]
Thanks

**Review by YANG-DB — APPROVED** [review]

File: test/panels_constants.tsx (hunk 1 above)
Comment (mengweieric) [reply test/panels_constants.tsx:7]: Yes but may not in this PR as this only deals with renaming. Verified in unit test run that there were no failing tests.

**Review by anirudha — APPROVED** [review]

**Comment by mengweieric** [comment]
backporting depends on #455

**Pull request merged**